### ISSUE TITLE
Picking up some items results in one no longer being displayed in inventory?

### ISSUE THREAD

**Issue by Resok** [CleverRaven/Cataclysm-DDA#40201]
# Describe the bug

When I pick up both a 'Finger Lighter CBM (Sterile)' and an 'anesthesia kit (3000/3000)', my inventory only displays the finger lighter CBM (Sterile), but it lists a '3' quantity next to it. The Anesthesia kit is not displayed at all.

# Steps To Reproduce

1. Load attached save (no mods)
2. Examine location to the SE
3. Pickup both items
4. Observe inventory screen showing - 

3 Finger Lighter CBMs  (sterile)

# Expected behavior

Both items and their 'status' should be listed separately

# Screenshots

![image](https://user-images.githubusercontent.com/5217673/81101706-61df1c00-8ec3-11ea-9ab8-0cc201b6aace.png)

![image](https://user-images.githubusercontent.com/5217673/81101778-791e0980-8ec3-11ea-983f-2823f4fc9374.png)

# Versions and configuration

- OS: Windows
    - OS Version: 10.0 1909
- Game Version: 0.E-1778-g781ca2e [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda]
]

# Additional context

When dropping the item listed as '3 Finger Lighter CBMs  (sterile)' - it drops both the CBM and the Anesthesia kit.

[Save.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/4582939/Save.zip)

**Comment by MamuelMamington** [comment]
I have seen a similar issue. This doesn't just create strange numbers on autopickup, but it can actually duplicate items. Tested it with .223 casings and was able to do so infinitely. Similarly, items disappear at random if they are of the same type as another (when 3 .223 magazines were in my inventory, 2 of them disappeared, but reappeared when the remaining one was dropped. strangely when I tried to reload, all three were still available in the menu)
